orcid-scheduler-web/src/main/java/org/orcid/scheduler/loader/cli/EmailDomainToRorLoader.java

```diff
@@ -0,0 +1,187 @@
+package org.orcid.scheduler.loader.cli;
+
+import java.io.FileReader;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+import org.orcid.core.common.manager.EmailDomainManager;
+import org.orcid.core.common.manager.impl.EmailDomainManagerImpl.STATUS;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.context.ApplicationContext;
+import org.springframework.context.support.ClassPathXmlApplicationContext;
+
+import com.fasterxml.jackson.databind.MappingIterator;
+import com.fasterxml.jackson.dataformat.csv.CsvMapper;
+import com.fasterxml.jackson.dataformat.csv.CsvParser;
+
+public class EmailDomainToRorLoader {
+
+    private static final Logger LOG = LoggerFactory.getLogger(EmailDomainToRorLoader.class);
+    
+    private String filePath;
+    private EmailDomainManager emailDomainManager;
+    List<List<String>> csvData;
+
+    Set<String> invalidDomains = new HashSet<String>();  
+    
+    Map<String, DomainToRorMap> map = new HashMap<String, DomainToRorMap>();
```

Comment:
Maybe csvData, invalidDomains, map can be private

Reply:
Well, they are not private just in case any other CLI will ever need it ... this is package level access, so, it is technically private to any class outside the `org.orcid.scheduler.loader.cli` package and could be used as public just by other CLIs. Since this is a good observation I will set them private